src/components/accommodation/AccommodationFormFields.js

```diff
@@ -5,43 +5,93 @@ import { FormikApiErrors } from "components/atoms/form/FormikApiErrors";
     static VACANCIES_TOTAL = "vacanciesTotal";
-    static VACANCIES_TAKEN = "vacanciesTaken";
-    static UUID = "uuid";
-    static VOLUNTEER_NAME = "volunteerName";
+    static VACANCIES_FREE = "vacanciesFree";
+    static DISABLED_PEOPLE_FRIENDLY = "disabledPeopleFriendly";
+    static LGBT_FRIENDLY = "lgbtFriendly";
+    static PARKING_PLACE = "parkingPlaceAvailable";
+    static EASY_AMBULANCE_ACCESS = "easyAmbulanceAccess";
+    // static DESCRIPTION = "description";
+    // static IS_VERIFIED = "isVerified";
+    // static HOST_EMAIL = "hostEmail";
+    // static HOST_NAME = "hostName";
+    // static HOST_PHONE = "hostPhone";
+    // static VOLUNTEER_NAME = "volunteerName";
+    // static CREATED_AT = 'createdAt';
+
+    static modelToFormMap = {
+        // Id
+        id: "id",
+        // Vacancies
+        [AccommodationFormFields.VACANCIES_TOTAL]: "vacanciesTotal",
+        [AccommodationFormFields.VACANCIES_FREE]: "vacanciesFree",
+        // Info
+        [AccommodationFormFields.STATUS]: "status",
+        [AccommodationFormFields.COMMENTS]: "staffComments",
+        // Address
+        [AccommodationFormFields.ADDRESS_CITY]: "city",
+        [AccommodationFormFields.ADDRESS_LINE]: "addressLine",
+        [AccommodationFormFields.ADDRESS_ZIP]: "zip",
+        [AccommodationFormFields.ADDRESS_VOIVODESHIP]: "voivodeship",
+        // Pets
+        [AccommodationFormFields.PETS_ALLOWED]: "petsAccepted",
+        [AccommodationFormFields.PETS_PRESENT]: "petsPresent",
+        // Accessibility
+        [AccommodationFormFields.DISABLED_PEOPLE_FRIENDLY]:
+            "disabledPeopleFriendly",
+        [AccommodationFormFields.LGBT_FRIENDLY]: "lgbtFriendly",
+        [AccommodationFormFields.PARKING_PLACE]: "parkingPlaceAvailable",
+        [AccommodationFormFields.EASY_AMBULANCE_ACCESS]: "easyAmbulanceAccess",
+    };
 
     /**
-     *
+     * Transform object from model to form object.
      * @param {Accommodation} accommodation
      * @return {*}
      */
-    static getInitialValues(accommodation) {
-        const fieldNames = Object.values(AccommodationFormFields);
-        const initialValues = pick(accommodation, fieldNames);
-
-        return accommodation.id
-            ? initialValues
-            : Object.assign(initialValues, {
-                  [AccommodationFormFields.VACANCIES_TOTAL]: 1,
-                  [AccommodationFormFields.VACANCIES_TAKEN]: 0,
-                  [AccommodationFormFields.IS_VERIFIED]: false,
-              });
+    static toForm(accommodation) {
+        // const fieldNames = Object.values(AccommodationFormFields);
+        // const initialValues = pick(accommodation, fieldNames);
+
+        // console.log("computing initial values, raw object = ", accommodation);
+        //
+        // console.log(
+        //     "computing initial values, voivodeship = ",
+        //     accommodation.voivodeship
+        // );
+
+        if (!accommodation.id) {
+            console.log(
```

Comment:
Let's issue warning here and add some meaningful prefix to the message, see other examples of console.log(). This will come in handy when we plug-in some error-collecting / logging library on the frontend